### DISCUSSION TITLE
Different icons in SFTP for symlinks (and related ideas)

### DISCUSSION THREAD

**Discussion by drmrbrewer** [isontheline/pro.webssh.net#819] [Ideas]
If a file or folder is actually a symlink, it would be great if a different folder icon could be used to make it clear... something [like this](https://visualpharm.com/free-icons/symlink%20directory-595b40b85ba036ed117de56f) just as an example:

![image](https://user-images.githubusercontent.com/9009792/218145594-4418ad8e-fe5f-4a1b-ba5c-fa5b45638812.png)

And IMHO it would be better not to show the full (linked) path in the folder name itself, and maybe include that info in a "properties" popup on long-press... otherwise it adds visual clutter.

Similar for symlinked files... a different icon to make it more clear that the file is a symlink to a file, not the file itself... something like:

![image](https://user-images.githubusercontent.com/9009792/218146425-98a81968-c48b-494e-ae2b-aa99ab3692ee.png)

And for symlinked files, in the file listing IMHO it would be better to see the size of the linked file (e.g. `1.11 GB`), not of the link itself (e.g. `42 B`), i.e. to make it appear more like it is an actual file (i.e. if you open it, you will open the `1.11 GB` file, not the `42 B` symlink).  Again, better not to show the full (linked) path in the file name itself, but more info about the actual symlink in a "properties" popup.

This is partly because in some of my filenames are very long, so I can't see the symlink arrow icon in the file/folder name unless I turn the screen to landscape (and I wouldn't normally want to do that), so it's not apparent which are symlinks and which are not.

**Comment by isontheline** [comment]
Thank you so much @drmrbrewer for your feedback 🙏

I'm opening an issue with the content of your discussion : https://github.com/isontheline/pro.webssh.net/issues/820
